.github/workflows/prbot.yml

```diff
@@ -0,0 +1,18 @@
+name: PR Robot
+on:
+- pull_request
```

Comment:
Would this trigger generate a report on pushes to an open PR? It would be helpful to see updated bundle size numbers,

Reply:
Yes, whenever a new change is pushed, this will re-run.

Reply:
Also related, you can open this view:

![image](https://user-images.githubusercontent.com/1591893/100926451-7adb9a00-34b1-11eb-91b6-9db1639f696a.png)

Click on PR Robot details -> re run if you need to manually re-trigger it for some reason